code/modules/cargo/supplypod.dm

```diff
@@ -266,7 +271,7 @@
 		playsound(get_turf(holder), openingSound, soundVolume, FALSE, FALSE) //Special admin sound to play
 	for (var/atom/movable/O in holder.contents) //Go through the contents of the holder
 		O.forceMove(T) //move everything from the contents of the holder to the turf of the holder
-	if (!effectQuiet && !openingSound && style != STYLE_SEETHROUGH) //If we aren't being quiet, play the default pod open sound
+	if (!openingSound && style != STYLE_SEETHROUGH) //If we aren't being stealthy, play the default pod open sound

```

Comment:
Why remove the effectQuiet check here?

Reply:
because i figured that if an admin set up a custom opening sound they want it to not be quiet